platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/util/function/EBiConsumer.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.util.function;
+
+import org.apache.commons.lang.Validate;
+
+/**
+ * The <code>EBiConsumer</code> interface expands on the {@link java.util.function.BiConsumer}
+ * interface to provide the ability to throw back exceptions.
+ * <p>
+ * <b>
+ * This code is experimental. While this interface is functional
+ * and tested, it may change or be removed in a future version of the
+ * library.
+ * </b>
+ * </p>
+ *
+ * @param <T> the type of the first argument to the operation
+ * @param <U> the type of the second argument to the operation
+ * @param <E> the type of exceptions that can be thrown by the operation
+ */
+@FunctionalInterface
+public interface EBiConsumer<T, U, E extends Throwable> {
```

Comment:
This general functional interface is a great candidate for a utility library like Apache. Using it here makes it so much simpler on clients since they do not have to worry about catching the IO exceptions that are thrown from dealing with output/input streams. Not having to do a try/catch just to re-throw an UncheckedIOException and have the framework unwrap that in every clients.